src/main/resources/db/changelog/ddl/0010_create_temporary_certificate_tables.xml

```diff
@@ -0,0 +1,113 @@
+            </column>
+            <column name="source_reference" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="application_reference" type="varchar(255)"/>
+            <column name="certificate_template_version" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issuing_authority" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="english_ero_id" type="uuid">
+                <constraints nullable="false"
+                             foreignKeyName="fk_temporary_certificate_english_ero_id"
+                             references="electoral_registration_office(id)"/>
+            </column>
+            <column name="issuing_authority_cy" type="varchar(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="welsh_ero_id" type="uuid">
+                <constraints nullable="true"
+                             foreignKeyName="fk_temporary_certificate_welsh_ero_id"
+                             references="electoral_registration_office(id)"/>
+            </column>
+
+            <column name="first_name" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="middle_names" type="varchar(255)"/>
+            <column name="surname" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="certificate_language" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="photo_location_arn" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="valid_on_date" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_id" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="version" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="nathan.russell@valtech.com" id="0010-EIP1-3610-create-temporary-certificate-status" context="ddl">
+        <createTable tableName="temporary_certificate_status">
```

Comment:
Much like with the `print_request` for the main cert where we have a collection of statuses to be able to show a timeline/history of events ... I'm thinking it might be prudent to do a similar thing here.

At the moment there is only 1 status we know about ("generated"), but it wouldn't surprise me if we were asked to track other events/statuses .... such as being printed, being collected/delivered to the elector, expiry etc

I know for AEDs they are currently talking about 2 statuses (generated and printed), so if that requirement comes out way we will be able to cater for it

Reply:
We'd possibly have a separate table for AEDs.  
Possibly this is  `temporary_certificate_status_history` with the current status stored directly on the `temporary_certificate`.

Reply:
Yes, `history` in the table name suffix. Like it 👍

Reply:
Is this going to be `temporary_certificate_status_hostory`?